clients/runtime/src/tests.rs

```diff
@@ -0,0 +1,134 @@
+#![cfg(test)]
+
+use std::{convert::TryInto, time::Duration};
+
+use sp_keyring::AccountKeyring;
+
+use primitives::{StellarPublicKeyRaw, TokenSymbol};
+
+use crate::{integration::*, FeedValuesEvent, OracleKey, VaultId};
+
+use super::{
+	CollateralBalancesPallet, CurrencyId, FixedPointNumber, FixedU128, OraclePallet,
+	SecurityPallet, StatusCode, Token, TryFromSymbol, VaultRegistryPallet,
+};
+
+const DEFAULT_TESTING_CURRENCY: CurrencyId = Token(TokenSymbol::KSM);
+const DEFAULT_WRAPPED_CURRENCY: CurrencyId = CurrencyId::AlphaNum4 {
+	code: *b"USDC",
+	issuer: [
```

Comment:
In literals such as this one, it is always useful to comment the human-readable issuer account for reference and future readers.